README.md

```diff
@@ -15,8 +15,9 @@ $ npm install --save oniyi-http-plugin-credentials
 const OniyiHttpClient = require('oniyi-http-client');
 const oniyiHttpPluginCredentials = require('oniyi-http-plugin-credentials');
 
-const clientOptions = {};
-const httpClient = new OniyiHttpClient(clientOptions);
+const httpClientParams = {
+  requestPhases: ['credentials'], // indicates that we want phase hook handler with name 'credentials' should be run in request phase list
```

Comment:
request phases MUST include the phase names `initial` and `final`, since `oniyi-http-client` uses [phaseList.zipMerge()](https://github.com/strongloop/loopback-phase/blob/master/lib/phase-list.js#L164) to merge whatever is provided here with the default phase names `['initial', 'final']`.
Without providing these two, you can not control in what order your custom phase is added